templates/base.html

```diff
@@ -1,4 +1,6 @@
 {% load staticfiles %} {% load static %} {% load group_tag %}
+{% load i18n %}
+
```

Comment:
Let's add trans tags on
-  `<meta name="description" content="Hikaya - Activity: We help organizations tell their stories through data" />` line 23
- `Objectives ` on line 158
- `Documents ` on line 188 
- `Dashboards ` on line 208
- `signed in as` on line 264
- `Profile ` on line 266
- `Organization Settings` on line 269
- `Add new Organization` on line 286
- `Sign out` on line 292

Please see, I think you missed this file on top-level templates.

Reply:
Thank you. Good catch.
Of all the top level templates, I miss the `base.html` :man_facepalming: